setup.py

```diff
@@ -39,6 +39,7 @@ def run(self):
     "dask_cloudprovider": ["dask_cloudprovider >= 0.2.0, < 1.0"],
     "dev": dev_requires + test_requires,
     "dropbox": ["dropbox ~= 9.0"],
+    "ge": ["great_expectations >= 0.11.0"],
```

Comment:
GE 0.11.0 had a small regression in it so I'd recommend 0.11.1+
```suggestion
    "ge": ["great_expectations >= 0.11.1"],
```